Content/error/404.md

```diff
@@ -0,0 +1,6 @@
+#  Error 404 - Page not found
+
+The page you are looking for might have been removed, had it's name changed or is temporarily unavailable.  
+Please try again later.  
+  
+[Return to the home page](https://vapor.codes)
```

Comment:
I'd say this should probably link to `/` to avoid any issues around redirects to the `www.` site etc

Reply:
```suggestion
[Return to the home page](/)
```